app/src/examples/RuntimeTests/tests/animations/withDecay/basic.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { View, StyleSheet } from 'react-native';
+import Animated, { useAnimatedStyle, withDecay, WithDecayConfig } from 'react-native-reanimated';
+import React from 'react';
+import { describe, test, render, wait } from '../../../ReanimatedRuntimeTestsRunner/RuntimeTestsApi';
+
+describe.only('withDecay animation, test various config', () => {
+  const DecayComponent = ({ config }: { config: WithDecayConfig }) => {
+    const animatedStyle = useAnimatedStyle(() => {
+      return {
+        left: withDecay(config),
+      };
+    });
+
+    return (
+      <View style={styles.container}>
+        <Animated.View style={[styles.animatedBox, animatedStyle]} />
+      </View>
+    );
+  };
+
+  test.each([
+    { velocity: 900 },
+    { velocity: 9, velocityFactor: 100 },
+    { velocity: 900, deceleration: 0.997 },
+    { velocity: 900, clamp: [0, 150] },
+    { velocity: 900, clamp: [0, 150], rubberBandEffect: true },
+    { velocity: 2000, clamp: [0, 150], rubberBandEffect: true },
+    { velocity: 2000, clamp: [0, 150], rubberBandEffect: true, rubberBandFactor: 2 },
+  ] as Array<WithDecayConfig>)('Config %p', async config => {
+    await render(<DecayComponent config={config} />);
+    await wait(3000);
+  });
+});
```

Comment:
There appears to be no `expect`-like statement, these tests will just run without any info whether they passed or not. Also, `3000ms` wait is quite a lot for each of the tests (maybe let's make it dependant on the config?)

Reply:
Oh, thank you so much! 😱 I'm always adding snapshots at the very end (because you have to replace them if you modify test case) and forgot.
Yes, `3000ms` is a lot, I'll make it quicker